docs/en/engines/table-engines/mergetree-family/mergetree.md

```diff
@@ -821,4 +823,51 @@ S3 disk can be configured as `main` or `cold` storage:
 
 In case of `cold` option a data can be moved to S3 if local disk free size will be smaller than `move_factor * disk_size` or by TTL move rule.
 
-[Original article](https://clickhouse.tech/docs/ru/operations/table_engines/mergetree/) <!--hide-->
+## Using HDFS for Data Storage {#table_engine-mergetree-hdfs}
+
+[HDFS](https://hadoop.apache.org/docs/r1.2.1/hdfs_design.html) is a distributed file system for remote data storage.
+
+`MergeTree` family table engines can store data to HDFS using a disk with type `HDFS`.
+
+Configuration markup:
+``` xml
+<yandex>
+    <storage_configuration>
+        <disks>
+            <hdfs>
+                <type>hdfs</type>
+                <endpoint>hdfs://hdfs1:9000/clickhouse/</endpoint>
+            </hdfs>
+            <hdd>
+                <type>local</type>
+                <path>/</path>
+            </hdd>
+        </disks>
+        <policies>
+            <hdfs>
+                <volumes>
+                    <main>
+                        <disk>hdfs</disk>
+                    </main>
+                    <external>
+                        <disk>hdd</disk>
+                    </external>
+                </volumes>
+            </hdfs>
+        </policies>
+    </storage_configuration>
+
+    <merge_tree>
+        <min_bytes_for_wide_part>0</min_bytes_for_wide_part>
+    </merge_tree>
+</yandex>
+```
+
+Required parameters:
+
+-   `endpoint` — HDFS endpoint URL in `path`. Endpoint URL should contain a bucket and root path to store data. The parameter is used to look up the HDFS configuration for retrieving files.
```

Comment:
In hdfs there are no buckets in the same way as in s3. Here should only mention `root path`. 
Also second sentence does not sound correct. This parameter is **not** to look up **configuration**, but to look up the **files** actually as they are stored by those paths.